app/presenters/openid_connect_user_info_presenter.rb

```diff
@@ -8,29 +8,55 @@ def initialize(identity)
   end
 
   def user_info
-    user = identity.user
-
-    {
+    info = {
       sub: identity.uuid,
       iss: root_url,
-      email: user.email,
+      email: identity.user.email,
```

Comment:
I would prefer not to delegate. When I see `identity.email` I immediately think there's a column called `email` on the `identity` model, then have to remind myself it's a shortcut.